packages/vaadin-grid/test/keyboard-navigation.test.js

```diff
@@ -1827,6 +1827,18 @@ describe('keyboard navigation', () => {
 
       expect(spy.callCount).to.equal(1);
     });
+
```

Comment:
Add a test that gets the relevant event context information (see the feature Document for details). Now that the event is fired on the `<vaadin-grid>` element directly, it's impossible to get any contextual information from it.

Reply:
Yeah should have tested for this. I've changed it to now dispatch the event from the cell (not cell content). You can now get the grid from the event target and getEventContext  on the event works correctly.